src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -201,6 +223,15 @@ internal abstract partial class BaseArgument : Operation, IArgument
         /// Conversion applied to the argument value after the invocation. Applicable only to VB Reference arguments.
         /// </summary>
         public abstract IOperation OutConversion { get; }
+        public override IEnumerable<IOperation> Children
+        {
+            get
+            {
+                yield return Value;
```

Comment:
our operation tree dumper we use in test uses the Walker to output operation tree structure, any change in order will cause test to fail. I made sure Children keep the order same as it was in the walker.